.github/workflows/go.yml

```diff
@@ -10,10 +10,25 @@ jobs:
 
   build:
     runs-on: ubuntu-latest
+
+    env:
+      DB_NAME: golang
+      DB_USER: root
+      DB_PASSWORD: 1234
+      DB_DSN: $DB_USER:$DB_PASSWORD@/$DB_NAME
+
     steps:
     - uses: actions/checkout@v3
 
+    - name: Set up Mysql
+      run : |
+          sudo /etc/init.d/mysql start
+          mysql -h 127.0.0.1 -e 'CREATE DATABASE ${{ env.DB_NAME }};' -u${{ env.DB_USER }} -p${{ env.DB_PASSWORD }}
```

Comment:
I think there should be spaces here:
`-u**_**${{ env.DB_USER }}`
`-p**_**${{ env.DB_PASSWORD }}`

Reply:
so executing of this requires sudo and no space after -p flag